nuget/AppCenterData.nuspec

```diff
@@ -41,9 +41,9 @@
     <file src="$android_dir$/Microsoft.AppCenter.Data.Android.Bindings.dll" target="lib/MonoAndroid403" /> -->
 
     <!-- iOS -->
-    <!-- <file src="$ios_dir$/Microsoft.AppCenter.Data.dll" target="lib/Xamarin.iOS10" />
+    <file src="$ios_dir$/Microsoft.AppCenter.Data.dll" target="lib/Xamarin.iOS10" />
```

Comment:
Missing a dependency group for Xamarin.iOS to add Newtonsoft json.
(No need in portable)